core/src/parser/utils.rs

```diff
@@ -216,6 +216,14 @@ pub enum RecordLastField {
     Ellipsis,
 }
 
+/// A single binding in a let block.
+#[derive(Clone, Debug)]
+pub struct LetBinding {
+    pub pat: Pattern,
+    pub annot: Option<LetMetadata>,
+    pub val: RichTerm,
```

Comment:
Nitpick, mostly for consistency in how we name public struct fields elsewhere (private items or local variables are different).

```suggestion
    pub pattern: Pattern,
    pub annot: Option<LetMetadata>,
    pub value: RichTerm,
```